Userland/Libraries/LibJS/Runtime/VM.h

```diff
@@ -264,7 +264,8 @@ class VM : public RefCounted<VM> {
 
     StackInfo m_stack_info;
 
-    HashMap<DeprecatedString, Symbol*> m_global_symbol_map;
+    // https://tc39.es/ecma262/#table-globalsymbolregistry-record-fields
```

Comment:
Is it intentional to only have the link and not the actual name in the comment?

Reply:
grey area as `GlobalSymbolMap` doesn't have its own section link in the spec, but name + fields table works. Thanks!